src/routes/Player/Player.js

```diff
@@ -585,11 +585,24 @@ const Player = ({ urlParams, queryParams }) => {
                 }
             }
         };
+        const onWheel = ({ deltaY }) => {
+            if (deltaY > 0) {
+                if (!subtitlesMenuOpen && !infoMenuOpen && !videosMenuOpen && !speedMenuOpen && !optionsMenuOpen && !statisticsMenuOpen && videoState.volume !== null) {
+                    onVolumeChangeRequested(videoState.volume - 5);
+                }
+            } else {
+                if (!subtitlesMenuOpen && !infoMenuOpen && !videosMenuOpen && !speedMenuOpen && !optionsMenuOpen && !statisticsMenuOpen && videoState.volume !== null) {
+                    onVolumeChangeRequested(videoState.volume + 5);
+                }
+            }
+        };
```

Comment:
Can be simplified to:

```suggestion
       const onWheel = ({ deltaY }) => {
    if (!subtitlesMenuOpen && !infoMenuOpen && !videosMenuOpen && !speedMenuOpen && !optionsMenuOpen && !statisticsMenuOpen && videoState.volume !== null) {
        onVolumeChangeRequested(deltaY > 0 ? videoState.volume - 5 : videoState.volume + 5);
    }
};
```

Or even shorter:
```ts
      // ...
      onVolumeChangeRequested(videoState.volume + (deltaY > 0 ? -5 : 5));
```